src/qml/RelationCombobox.qml

```diff
@@ -302,25 +321,22 @@ Item {
                 anchors.fill: parent
                 propagateComposedEvents: true
 
-                onClicked: { mouse.accepted = false; }
-                onPressed: {
-                    forceActiveFocus();
-                    mouse.accepted = false;
+                onClicked: (mouse) => { mouse.accepted = false }
```

Comment:
A bit weird the variable is called `mouse` in touch environment... I guess all the docs and examples are already having `mouse` so changing on our end is not smart either.